__tests__/app.test.js

```diff
@@ -129,6 +129,83 @@ describe("Using app.js to run the database of NC-games", () => {
         });
     });
   });
-  //Head describe DELETE later!
+  describe("6. PATCH/api/reviews/:reviews_id", () => {
+    test("201: response with a board game object with a updated vote", () => {
+      const updateVote = { votes: 2 };
+      const review_id = 2;
+      return request(app)
+        .post(`/api/reviews/${review_id}`)
+        .send(updateVote)
+        .expect(201)
+        .then(({ body }) => {
+          expect(body.updatedData.votes).toBe(7);
+        });
```

Comment:
Please make an assertion that the whole of the data object that comes back has the structure we expect (e.g. all the keys are there as expected)

Reply:
Oh man... Okay all test now have propertys for expected results